src/main/java/org/quantumbadger/redreader/adapters/MainMenuListingManager.java

```diff
@@ -266,30 +276,40 @@ public MainMenuListingManager(
 		}
 
 		if(!user.isAnonymous()) {
-			if (PrefsUtility.pref_show_multireddit_main_menu(
+			if(PrefsUtility.appearance_navigation_type(
 					activity,
-					PreferenceManager.getDefaultSharedPreferences(activity))) {
+					PreferenceManager.getDefaultSharedPreferences(activity)).equals("single_list")) {
+
+				if (PrefsUtility.pref_show_multireddit_main_menu(
+						activity,
+						PreferenceManager.getDefaultSharedPreferences(activity))) {
 
-				showMultiredditsHeader(activity);
+					showMultiredditsHeader(activity);
 
-				final LoadingSpinnerView multiredditsLoadingSpinnerView = new LoadingSpinnerView(activity);
-				final int paddingPx = General.dpToPixels(activity, 30);
-				multiredditsLoadingSpinnerView.setPadding(paddingPx, paddingPx, paddingPx, paddingPx);
+					final LoadingSpinnerView multiredditsLoadingSpinnerView = new LoadingSpinnerView(activity);
+					final int paddingPx = General.dpToPixels(activity, 30);
+					multiredditsLoadingSpinnerView.setPadding(paddingPx, paddingPx, paddingPx, paddingPx);
 
-				final GroupedRecyclerViewItemFrameLayout multiredditsLoadingItem
-						= new GroupedRecyclerViewItemFrameLayout(multiredditsLoadingSpinnerView);
-				mAdapter.appendToGroup(GROUP_MULTIREDDITS_ITEMS, multiredditsLoadingItem);
+					final GroupedRecyclerViewItemFrameLayout multiredditsLoadingItem
+							= new GroupedRecyclerViewItemFrameLayout(multiredditsLoadingSpinnerView);
+					mAdapter.appendToGroup(GROUP_MULTIREDDITS_ITEMS, multiredditsLoadingItem);
+				}
 			}
 		}
 
 		if (PrefsUtility.pref_show_subscribed_subreddits_main_menu(
```

Comment:
I take it the drawer ignores the main menu item preferences? This could be confusing for the user.